package.json

```
@@ -66,45 +65,42 @@
     "pkg-conf": "^3.1.0",
     "resolve": "^1.19.0",
     "semver": "^7.3.4",
-    "update-dotenv": "^1.1.1",
-    "uuid": "^8.3.2"
+    "update-dotenv": "^1.1.1"
   },
   "devDependencies": {
-    "@octokit/webhooks-examples": "^4.0.0",
-    "@octokit/webhooks-methods": "^3.0.0",
-    "@octokit/webhooks-types": "^4.0.0",
+    "@octokit/webhooks-examples": "^7.0.0",
+    "@octokit/webhooks-methods": "^4.0.0",
+    "@octokit/webhooks-types": "^7.0.0",
     "@tsconfig/node10": "^1.0.7",
     "@types/eventsource": "^1.1.5",
-    "@types/jest": "^26.0.18",
+    "@types/jest": "^29.5.5",
     "@types/js-yaml": "^3.12.5",
-    "@types/jsonwebtoken": "^8.5.0",
-    "@types/node": "^16.0.0",
-    "@types/readable-stream": "^2.3.9",
+    "@types/node": "^18.18.1",
     "@types/resolve": "^1.17.1",
     "@types/semver": "^7.3.4",
     "@types/supertest": "^2.0.10",
-    "@types/uuid": "^8.3.0",
     "body-parser": "^1.19.0",
     "bottleneck": "^2.19.5",
     "connect-sse": "^1.2.0",
     "execa": "^5.0.0",
+    "fetch-mock": "^9.11.0",
```

There are some issues with `fetch-mock` and using the Fetch API, you will have to use this fork until it is merged upstream
```suggestion
    "fetch-mock": "npm:@gr2m/fetch-mock@9.11.0-pull-request-644.1",
```

---

done